### PR TITLE
Bump lower bound on cardano-crypto-class

### PR DESCRIPTION
**Pull request by michaelpj**
This is the version that actually added the BLST support.

### CODE REVIEW

File: plutus-core/plutus-core.cabal [hunk 1]
```diff
@@ -263,7 +263,7 @@ library
     , bimap
     , bytestring
     , cardano-crypto
-    , cardano-crypto-class        ^>=2.1.0.2
+    , cardano-crypto-class        ^>=2.1.2
```
Comment (kwxm) [inline plutus-core/plutus-core.cabal:266]: I was tempted to leave a massive multi-page review just to be annoying, but OK, I'm willing to believe this.

**Review by kwxm — APPROVED** [review]

File: plutus-core/plutus-core.cabal (hunk 1 above)
Comment (michaelpj) [reply plutus-core/plutus-core.cabal:266]: My evidence is https://github.com/input-output-hk/cardano-haskell-packages/pull/291

Comment (kwxm) [reply plutus-core/plutus-core.cabal:266]: So why did it work?  It just happened to find the most recent version?

Comment (michaelpj) [reply plutus-core/plutus-core.cabal:266]: Yep! The issue was that if you don't have blst installed, cabal will just go looking for a version that doesn't require it, and this lets it pick an old version, which doesn't compile for other reasons. So this mostly just forces the solver error about missing blst.

**Pull request merged**